### ISSUE TITLE
PYTHONPATH appears in dir when venv is active

### ISSUE THREAD

**Issue by brianmoran** [bhilburn/powerlevel9k#650]
When I activate a virtualenv, the dir part of the prompt is replaced with **~PYTHONPATH**

.zshenv
```
export POWERLEVEL9K_MODE='nerdfont-complete'
export POWERLEVEL9K_PROMPT_ON_NEWLINE=true
export POWERLEVEL9K_DISABLE_RPROMPT=true
export POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(os_icon dir virtualenv vcs)
export POWERLEVEL9K_VCS_GIT_GITHUB_ICON=$'\uf113'
export POWERLEVEL9K_VCS_HIDE_TAGS=true
export POWERLEVEL9K_VCS_CLEAN_FOREGROUND='green'
export POWERLEVEL9K_VCS_CLEAN_BACKGROUND='black'
export POWERLEVEL9K_VCS_UNTRACKED_FOREGROUND='yellow'
export POWERLEVEL9K_VCS_UNTRACKED_BACKGROUND='black'
export POWERLEVEL9K_VCS_MODIFIED_FOREGROUND='red'
export POWERLEVEL9K_VCS_MODIFIED_BACKGROUND='black'
```

<img width="1007" alt="screen shot 2017-10-17 at 11 43 02 am" src="https://user-images.githubusercontent.com/5261397/31674671-65cef7c0-b330-11e7-9b07-748617d4ec18.png">

**Comment by bhilburn** [comment]
@brianmoran - Well, that sounds pretty strange, hah. Can you post a couple more screenshots showing what you mean, exactly?

**Comment by brianmoran** [comment]
@bhilburn - Here is another screenshot.

Steps to reproduce:

1. Go to directory with a virtual environment.
2. Activate virtual environment
3. Directory path is replaced with ~PYTHONPATH

I'm using prezto and pyenv if that makes any difference.

![screen shot 2017-10-18 at 7 35 08 am](https://user-images.githubusercontent.com/5261397/31716812-71e6a65a-b3d7-11e7-9b50-b519a84a19c9.png)

**Comment by shibumi** [comment]
I can't reproduce this with my setup:
![](https://paste.archlinux.de/xW6f/)

But I have another bug. When I switch in a different directory (for example /) I also see the `virtualenv` segment in my shell.. Isn't it supposed to be only visible when I am in the project directory? Or did I get that `virtualenv` stuff wrong?

**Comment by brianmoran** [comment]
@shibumi - Your venv is active until you close your shell or run **deactivate**

**Comment by brianmoran** [comment]
@bhilburn - I was able to hack this locally to fix the prompt. The hack is in left_prompt_segment. Basically I check if $5 == ~PYTHONPATH and echo out $PWD instead. It looks like prompt_virtualenv is passing in the the basename of the $VIRTUAL_ENV variable which is causing $5 to be ~PYTHONPATH in left_prompt_segment


<img width="613" alt="screen shot 2017-10-24 at 3 13 09 pm" src="https://user-images.githubusercontent.com/5261397/31963228-ffc94ac2-b8cd-11e7-87a2-87140efc0040.png">

![screen shot 2017-10-24 at 3 17 06 pm](https://user-images.githubusercontent.com/5261397/31963381-70ab088e-b8ce-11e7-989b-aa2e29661fde.png)

**Comment by bhilburn** [comment]
@brianmoran - Ah, thanks for tracking down the problem, and glad you got it working!

Do you have any interest in submitting a PR to fix this? We are always interested in submissions from new contributors =)

**Comment by brianmoran** [comment]
@bhilburn I found another bug related to ~PYTHONPATH in the dir. I will fix and submit a PR. Thanks!

**Comment by brianmoran** [comment]
Before
![before](https://user-images.githubusercontent.com/5261397/32117816-a1275a02-bb1d-11e7-9dad-c8377de54bf1.gif)

After
![after](https://user-images.githubusercontent.com/5261397/32117819-a4859394-bb1d-11e7-8062-65f3836fffa9.gif)

**Comment by bhilburn** [comment]
Fixed by @brianmoran's awesome PR #660
